module/motion/KinematicsConfiguration/src/KinematicsConfiguration.cpp

```diff
@@ -148,14 +135,28 @@ namespace motion {
     }
 
     void KinematicsConfiguration::configureMassModel(KinematicsModel& model, const YAML::Node& objMassModel) {
+        model.massModel.head        = convert<double, 4>(objMassModel["particles"]["head"].as<arma::vec4>());
```

Comment:
Don't we have a .as for Eigen types directly?

Reply:
We did at one point, during the massive armadillo purge, alas much was lost during there trying times